test/e2e/singlecluster/e2e_test.go

```diff
@@ -153,7 +155,7 @@ var _ = ginkgo.Describe("Kueue", func() {
 			ginkgo.By("Create the pebuilt workload and the job adopting it", func() {
 				sampleJob = (&testingjob.JobWrapper{Job: *sampleJob}).
 					Label(constants.PrebuiltWorkloadLabel, "prebuilt-wl").
-					Image("gcr.io/k8s-staging-perf-tests/sleep:v0.1.0", []string{"5s"}).
+					Image("gcr.io/k8s-staging-perf-tests/sleep:v0.1.0", []string{"1ms"}).
```

Comment:
I think this test does not benefit that much from it because it deletes (kills) all its pods once done, and uses backoffLimit=0. So, for this binary we don't need a binary that ends on its own.

Reply:
I agree. I'll replace the image with the `pause` one since it doesn't end